src/room/GroupCallView.tsx

```diff
@@ -135,44 +135,41 @@ export const GroupCallView: FC<Props> = ({
   latestMuteStates.current = muteStates;
 
   useEffect(() => {
-    const defaultDeviceSetup = async (
-      requestedDeviceData: JoinCallData,
-    ): Promise<void> => {
+    const defaultDeviceSetup = async ({
+      audioInput,
+      videoInput,
+    }: JoinCallData): Promise<void> => {
       // XXX: I think this is broken currently - LiveKit *won't* request
       // permissions and give you device names unless you specify a kind, but
       // here we want all kinds of devices. This needs a fix in livekit-client
       // for the following name-matching logic to do anything useful.
       const devices = await Room.getLocalDevices(undefined, true);
-      const { audioInput, videoInput } = requestedDeviceData;
-      if (audioInput === null) {
-        latestMuteStates.current!.audio.setEnabled?.(false);
-      } else {
```

Comment:
Why can we remove this call?
Do we know the default is false anyways in this situation.

I feel like the ref pointer here might also deserve a comment while we are at it:
```
  const latestMuteStates = useRef<MuteStates>();
  latestMuteStates.current = muteStates;
```

Explaining what is stored in latestMuteStates and why we not use `muteStates` directly.

Reply:
I'm not sure we can remove it. We don't know that the default is false if the input isn't matched.

I've added a TODO note for the comment. I don't actually know why we are using a ref.